src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -172,7 +172,7 @@ func NewClient(ctx context.Context, opts ...option.ClientOption) (*Client, error
         client: librarypb.NewLibraryServiceClient(conn),
         labelerClient: taggerpb.NewLabelerClient(conn),
     }
-    c.SetGoogleClientInfo("gapic", version.Repo)
+    c.SetGoogleClientInfo("", "")
```

Comment:
This is called from the "constructor" of the client. I think this is as close to default as we'll get.